Cargo.toml

```diff
@@ -14,6 +14,7 @@ serde_derive = "1.0"
 serde_yaml = "0.7"
 structopt = "0.2"
 uuid = { version = "0.6", features = ["v4"] }
+yaml-rust = "0.4.3"
 
 [dev-dependencies]
 tempdir = "0.3.7"
```

Comment:
You can probably use `tempdir` for testing this (write out a test yaml file).

A particularly interesting test case is when you have a director structure like

```
- floki.yaml
- config.yaml
- some_dir/
  - blah
```

and run `floki` from `some_dir`. Does it locate the `config.yaml` properly?

Reply:
No it doesn't locate `config.yaml`! I wasn't aware that floki could be run from a subdirectory...

I've assumed that the image.yaml.file path should be relative to the floki.yaml path (seems reasonable -right?), so I've fixed this by prepending image.yaml.file with the path to floki.yaml's directory.

Reply:
It's a new feature, so it makes sense you wouldn't expect it!

Your assumptions sound reasonable, and align with mine.